comps/reranks/README.md

```diff
@@ -0,0 +1,39 @@
+# ReRank Microservice
+
+## Introduction
+
+Provides functionality for reranking outputs.
```

Comment:
a little bit confusing. Reranker is designed to provide fine-grained ranking for the result of retrievers.